src/app/caesar-portal/group-list/create-edit-dialog/create-edit-dialog.component.html

```diff
@@ -0,0 +1,164 @@
+               </li>
+             </ul>
+           </div>
+         </div>
+       </div>
+     </div>
+    </div>
+    <div class="row items-row">
+      <div class="col-md-12">
+        <div class="col-md-6">
+          <div class="row">
+            <div class="col-md-12">
+              <div class="row">
+                <div class="col-md-4">
+                  Direction:
+                </div>
+                <div class="col-md-8">
+                  <div dropdown>
+                    <button dropdownToggle class="btn btn-dropdown dropdown-toggle" type="button">
+                      {{selectedDirection?.name}}
+                      <span class="caret"></span>
+                    </button>
+                    <ul *dropdownMenu class="dropdown-menu" role="menu">
+                      <li role="menuitem" *ngFor="let direction of directions">
+                        <a class="dropdown-item" href="#" (click)="changeSelectedDirection(direction)">{{direction.name}}</a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-12">
+              <div class="row">
+                <div class="col-md-4">
+                  Location:
+                </div>
+                <div class="col-md-8">
+                  <div dropdown [isDisabled]="currentUser?.role?.id !== allowedRole">
+                    <button dropdownToggle class="btn btn-dropdown dropdown-toggle" type="button">
+                      {{selectedLocation?.name}}
+                      <span class="caret"></span>
+                    </button>
+                    <ul *dropdownMenu class="dropdown-menu" role="menu">
+                      <li role="menuitem" *ngFor="let location of locations">
+                        <a class="dropdown-item" href="#" (click)="changeSelectedLocation(location)">{{location.name}}</a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="row">
+            <div class="col-md-12">
+              <div class="row">
+                <div class="col-md-4">
+                  Start date:
+                </div>
+                <div class="col-md-8">
+                  <div class="col-md-9 selected-date">{{dateStart | date: 'MM / dd / y'}}</div>
+                  <button class="btn btn-datepicker" (click)="toggleDatepicker('isShownDatepickerStart')"><i class="glyphicon glyphicon-calendar"></i></button>
+                  <div class="datepicker-popup" [hidden]="!isShownDatepickerStart">
+                    <datepicker [(ngModel)]="dateStart" [startingDay]="1" (selectionDone)="selectionDone('isShownDatepickerStart')" (activeDateChange)="calculateFinishDate()"></datepicker>
```

Comment:
too long lines is bad - make each property from the new line to improve readability. Pls do the same for all long lines